src/classes/HH_ManageHH_TEST.cls

```diff
@@ -88,4 +88,37 @@ private class HH_ManageHH_TEST {
         
         Test.stopTest();
     }
+
+    /*********************************************************************************************************
+    * @description Tests opening the ManageHouseholdUI page on an new HH Object
+    */
+    public static testMethod void testNewHHObject() { 
+        
+        npo02__Household__c hh = new npo02__Household__c(Name = 'test_hh');
+        insert hh;
```

Comment:
@davidhabib just curious why the test is creating the test hh, it doesn't appear to be used anywhere?